models/sipnet/R/met2model.SIPNET.R

```diff
@@ -21,8 +21,8 @@
 ##' @param end_date the end date of the data to be downloaded (will only use the year part of the date)
 ##' @param overwrite should existing files be overwritten
 ##' @param verbose should the function be very verbose
-met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date, overwrite=FALSE,verbose=FALSE,...){
-print("START met2model.SIPNET")
+met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date, ..., overwrite=FALSE,verbose=FALSE){
```

Comment:
This is just a 'catch all' while we're refining convert.inputs, which has the capacity to both take optional arguments itself and to pass on optional arguments to the conversion functions. This also ensures that if convert.inputs passes on an optional argument that one model needs, but another doesn't need/use, it doesn't crash the model converter that doesn't need that argument.